src/types/createCampaignCommon.ts

```diff
@@ -24,6 +24,24 @@ export type ImageSizes = {
   height: number
 }
 
+type BannerFormatTypes =
```

Comment:
`BannerFormatTypes`? - isn't it more like `SupplyStats` as there are bid floors as well. `BannerFormatTypes` does not corespond to the data in the type

Reply:
you shoud change the names of of the other places this is used, as at smoe point we will use the bid floors for something and the code will look like ```bidFloor: bannerSizes. appBidFloors```